site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -283,7 +285,7 @@ RUNNING: agones-test
 ERROR: pods "agones-test" already exists
 Error: 1 test(s) failed
 ```
-That mean that you skiped `--cleanup` flag and you should either delete `agones-test` pod manually or run with the same test `helm test my-release --cleanup` two more times.
+That mean that you skipped `--cleanup` flag and you should either delete `agones-test` pod manually or run with the same test `helm test my-release --cleanup` two more times.
```

Comment:
nit: That means that you skipped the `--cleanup` flag ... delete the `agones-test` pod ...

Reply:
Fixed!